### ISSUE TITLE
nvimcom does not install on windows

### ISSUE THREAD

**Issue by FvD** [jalvesaq/Nvim-R#43]
Trying to run Nvim-R on Windows 10 (starting with `\rf`) yields the error message:

```
* installing to library 'C:/Users/frans/Documents/R/win-library/3.2'
* installing *source* package 'nvimcom' ...
** libs

*** arch - i386
Warning: running command 'make -f "Makevars.win" -f "C:/PROGRA~1/R/R-32~1.4RE/etc/i386/Makeconf" -f "C:/PROGRA~1/R/R-32~1.4RE/share/make/winshlib.mk" SHLIB="nvimcom.dll" OBJECTS="nvimcom.o"' had status 127
ERROR: compilation failed for package 'nvimcom'
* removing 'C:/Users/frans/Documents/R/win-library/3.2/nvimcom
```

**Comment by jalvesaq** [comment]
There were some important changes from Vim-R-plugin to Nvim-R. This is from the plugin's documentation:

> On Windows, you have to install Rtools to be able to build the package:
> https://cran.r-project.org/bin/windows/Rtools/
> If you can't install Rtools, you may want to install the windows binary
> `nvimcom.zip` released with Nvim-R:
> https://github.com/jalvesaq/Nvim-R/releases

**Comment by jalvesaq** [comment]
The Cream project released "Vim without Cream" 7.4.1641.
You can now install Nvim-R 0.9.3 + nvimcom_0.9-14.zip from https://github.com/jalvesaq/Nvim-R/releases
However, to follow the development version of Nvim-R, you have to install Rtools because eventually I may need to change something in nvimcom, changing its version number triggering its automatic update (compilation).
